Tasks/VsTest/VSTest.ps1

```diff
@@ -38,9 +43,17 @@ if($testAssemblyFiles)
 {
     Write-Verbose "Calling Invoke-VSTest for all test assemblies"
     $timeline = Start-Timeline -Context $distributedTaskContext
-	$cwd = Get-Location  
```

Comment:
This may not be a good working directory, I am not quite sure what Get-Location is going to return in the context of running this task in the agent.